quarkus/tests/integration/src/test/java/org/keycloak/it/storage/database/dist/MariaDBDistTest.java

```diff
@@ -17,22 +17,12 @@
 
 package org.keycloak.it.storage.database.dist;
 
-import org.keycloak.it.junit5.extension.CLIResult;
 import org.keycloak.it.junit5.extension.CLITest;
 import org.keycloak.it.junit5.extension.WithDatabase;
-import org.keycloak.it.storage.database.BasicDatabaseTest;
+import org.keycloak.it.storage.database.MariaDBTest;
 
 @CLITest
 @WithDatabase(alias = "mariadb")
-public class MariaDBDistTest extends BasicDatabaseTest {
+public class MariaDBDistTest extends MariaDBTest {
```

Comment:
Just out of curiosity, what is the difference between `MariaDBDistTest` and `MariaDBTest`. They seem identical now (incl. annotations).

Reply:
One runs in the same JVM as tests, the other is running the dist.

JVM tests are easier to develop/test/debug. And dist tests are more a E2E test.

We should have more dist tests.

Reply:
tbh i am more into having more CLITests, as they are whiteboxed (as you say, easily debuggable etc). but that's sadly most of the time not possible bc. they are not playing well when a `build` is invoked and thus another java process spawns, so we rely on blackbox tests for that one.